src/app/containers/FrontPageStoryRows/__snapshots__/index.test.jsx.snap

```diff
@@ -571,7 +571,7 @@ exports[`FrontPageStoryRows Container - snapshots LeadingRow - rtl 1`] = `
             >
               <span
                 class="c8"
-                lang="en-GB"
+                lang="ar"
```

Comment:
Is this correct?

Reply:
Sorry @greenc05 this is still work in progress